File: src/common/slow_to_check_logging.ml
(*
 * Copyright (c) Meta Platforms, Inc. and affiliates.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 *)

type t = {
  slow_files_logging_internal: float option;
  slow_components_logging_threshold: float option;
  slow_expressions_logging_threshold: float option;
}

let default =
  {
    slow_files_logging_internal = None;
    slow_components_logging_threshold = None;
    slow_expressions_logging_threshold = None;
  }

File: rust_port/crates/flow_common/src/slow_to_check_logging.rs
/*
 * Copyright (c) Meta Platforms, Inc. and affiliates.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 */

#[derive(Debug, Clone, Copy, Default, serde::Serialize, serde::Deserialize)]
pub struct SlowToCheckLogging {
    pub slow_files_logging_internal: Option<f64>,
    pub slow_components_logging_threshold: Option<f64>,
    pub slow_expressions_logging_threshold: Option<f64>,
}
